lib/controller.js

```diff
@@ -300,21 +317,24 @@ export class AdvancedOpenFileController {
         }
 
         this.view.setCursorIndex(index);
+        this.stopWaitingNewFileConfirmation();
     }
 
     moveCursorTop() {
         this.view.setCursorIndex(0);
+        this.stopWaitingNewFileConfirmation();
     }
 
     moveCursorBottom() {
         this.view.setCursorIndex(this.view.pathListLength() - 1);
+        this.stopWaitingNewFileConfirmation();
```

Comment:
Please let me know if you see a better way than repeating this on every action

Reply:
I can't think of a better way that doesn't involve some significant refactoring in how we handle events. Ohhhhhhh welllllll